src/Components/Navbar/Navbar.js

```diff
@@ -1,30 +1,60 @@
-import './Navbar.css'
-import Tomatillo from '../../images/Tomatillo.png'
+import React, { useState, useEffect } from 'react';
+import './Navbar.css';
+import Tomatillo from '../../images/Tomatillo.png';
 import { NavLink } from 'react-router-dom';
 
-const Navbar = ({search, searchFilter}) => {
+const Navbar = ({ search, searchFilter }) => {
+  // Logic for setting the day message
+  const [dayMessage, setDayMessage] = useState('');
+  useEffect(() => {
+    const time = new Date();
+    const day = time.toLocaleString('en-US', { weekday: 'long' });
+    const morning = time.getHours() >= 6 && time.getHours() <= 12;
+
+    let message;
+
+    if (day.toLowerCase() === 'monday') {
```

Comment:
Consider using a switch statement here! A bit more readable than a much of if blocks.